app.js

```diff
@@ -25,4 +29,12 @@ stopButton.addEventListener("click", () => {
 resetButton.addEventListener("click", () => {
     clock.textContent = 0;
     seconds = 0;
-});
\ No newline at end of file
+    lapsUl.textContent = '';
+});
+
+lapButton.addEventListener('click', () => {
```

Comment:
your lap button functionality doesn't meet the criteria. Here you're just appending the counter value at time of press, you need to calc the diff and display that lap time.